### PR TITLE
TAN-19: component tag

### PR DESCRIPTION
**Pull request by dan-kiss-dev-this**

### CODE REVIEW

**Review by illepic — APPROVED** [review]

File: components/tag.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+type TagProps = {
+  text: String;
+  selected?: boolean;
+};
+
+export default function Tag({ text, selected = false }: TagProps) {
+  const [isSelected, setIsSelected] = useState(selected);
+
+  return (
+    <div className="py-16">
```
[the remark this answers is not in the document]
Comment (dan-kiss-dev-this) [reply components/tag.tsx:14]: done

File: components/tag.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <Switch
+        checked={isSelected}
+        onChange={setIsSelected}
+        className={clsx(
+          "inline-block text-black font-bold py-4 px-5 rounded-full",
+          {
+            "bg-tangie-bluePurple-500 border-2 box-border border-transparent text-white":
```
[the remark this answers is not in the document]
Comment (dan-kiss-dev-this) [reply components/tag.tsx:21]: done

File: components/tag.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (dan-kiss-dev-this) [reply components/tag.tsx:14]: done

File: tailwind.config.js [hunk 3]
```diff
@@ -11,6 +11,9 @@ module.exports = {
           mint: {
             500: "#13EAA8",
           },
+          bluePurple: {
```
[the remark this answers is not in the document]
Comment (dan-kiss-dev-this) [reply tailwind.config.js:14]: done

[the remark this answers is not in the document]
Comment (dan-kiss-dev-this) [reply tailwind.config.js]: done

**Review by illepic — APPROVED** [review]

**Pull request merged**